python/ray/tests/test_output.py

```diff
@@ -328,6 +328,7 @@ def verify():
     wait_for_condition(verify)
 
 
+@pytest.mark.skipif(sys.platform == "win32", reason="Failing on Windows.")
```

Comment:
Hmm can we just fix it? I have impression it should be a simple fix?

Reply:
it's related to running subprocess in windows - i tried [here](https://github.com/ray-project/ray/pull/38225), but the dev experience is a bit long-winded, i cannot test it locally but have to rely on CI runs.